integration-test/src/it/relocation-plugin/pom.xml

```diff
@@ -59,7 +60,7 @@
     <plugins>
       <plugin>
         <groupId>com.soebes.maven.plugins</groupId>
-        <artifactId>iterator-maven-plugin</artifactId>
+        <artifactId>repeater-maven-plugin</artifactId>
```

Comment:
Why this change. It looks like iterator-maven-plugin does exist, but no repeater-maven-plugin exists: https://search.maven.org/search?q=g:com.soebes.maven.plugins.

Reply:
Er....because we are relocating as a test?

Reply:
Oops, I missed there were two different relocations in this test.